pipelines/ops/ml-mongo-dump/Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
+        String ANSI_NORMAL = "\u001B[0m"
+        String ANSI_BOLD = "\u001B[1m"
+        String ANSI_RED = "\u001B[31m"
+        String ANSI_YELLOW = "\u001B[33m"
+
+        ansiColor('xterm') {
+            cleanWs()
+            stage('Checkout') { 
+                checkout scm            
+            }
+
+            ip=params.ip
+            database=params.database
+            
+            stage('Backup mongo single DB') {
+                
+                sh '''
+                echo $ip
+                echo $database
+                echo $WORKSPACE
+                mongodump --host="$ip" --port=27017 --out=$database --db=$database
```

Comment:
Please add the dependency in `jenkins-server-setup.sh` file to install `mongodump` command on the jenkins vm.